xbmc/addons/interfaces/General.cpp

```diff
@@ -409,6 +409,47 @@ int Interface_General::get_global_idle_time(void* kodiBase)
   return g_application.GlobalIdleTime();
 }
 
+void Interface_General::kodi_version(void* kodiBase, char** compile_name, int* major, int* minor, char** revision, char** tag, char** tagversion)
```

Comment:
Is moving this function necessary? If it is, this is the wrong place.

Reply:
Same as https://github.com/xbmc/xbmc/pull/16455/files#r311907505